scripts/microbe_stage/organelle.as

```diff
@@ -858,6 +863,12 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
         }
     }
 
+    int cachedHexCount {
```

Comment:
I think it would be better if there was no this method and instead everything would use a new method `organelle.getHexCount()`